[2024-06-28T10:12:01.989+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-28T10:12:02.017+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: demo_dag_for_alerting_apis_final.check_previous_run_status scheduled__2024-06-28T10:11:00+00:00 [queued]>
[2024-06-28T10:12:02.029+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: demo_dag_for_alerting_apis_final.check_previous_run_status scheduled__2024-06-28T10:11:00+00:00 [queued]>
[2024-06-28T10:12:02.029+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-06-28T10:12:02.048+0000] {taskinstance.py:2330} INFO - Executing <Task(BranchPythonOperator): check_previous_run_status> on 2024-06-28 10:11:00+00:00
[2024-06-28T10:12:02.058+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=162) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-28T10:12:02.059+0000] {standard_task_runner.py:63} INFO - Started process 163 to run task
[2024-06-28T10:12:02.060+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'demo_dag_for_alerting_apis_final', 'check_previous_run_status', 'scheduled__2024-06-28T10:11:00+00:00', '--job-id', '4961', '--raw', '--subdir', 'DAGS_FOLDER/demo_dag_for_alerting_apis.py', '--cfg-path', '/tmp/tmp5fcv9g1i']
[2024-06-28T10:12:02.061+0000] {standard_task_runner.py:91} INFO - Job 4961: Subtask check_previous_run_status
[2024-06-28T10:12:02.116+0000] {task_command.py:426} INFO - Running <TaskInstance: demo_dag_for_alerting_apis_final.check_previous_run_status scheduled__2024-06-28T10:11:00+00:00 [running]> on host 2552e7b36400
[2024-06-28T10:12:02.261+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='demo_dag_for_alerting_apis_final' AIRFLOW_CTX_TASK_ID='check_previous_run_status' AIRFLOW_CTX_EXECUTION_DATE='2024-06-28T10:11:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-28T10:11:00+00:00'
[2024-06-28T10:12:02.262+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-28T10:12:02.284+0000] {baseoperator.py:399} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-06-28T10:12:02.285+0000] {python.py:237} INFO - Done. Returned value was: start_run
[2024-06-28T10:12:02.285+0000] {branch.py:36} INFO - Branch into start_run
[2024-06-28T10:12:02.285+0000] {skipmixin.py:178} INFO - Following branch start_run
[2024-06-28T10:12:02.304+0000] {skipmixin.py:246} INFO - Skipping tasks [('skip_run', -1)]
[2024-06-28T10:12:02.339+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-28T10:12:02.360+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=demo_dag_for_alerting_apis_final, task_id=check_previous_run_status, run_id=scheduled__2024-06-28T10:11:00+00:00, execution_date=20240628T101100, start_date=20240628T101202, end_date=20240628T101202
[2024-06-28T10:12:02.394+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-28T10:12:02.431+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-28T10:12:02.433+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
